tests/cypress/support/e2e.ts

```diff
@@ -33,6 +33,7 @@ declare global {
       checkGitRepoStatus(repoName: string, bundles?: string, resources?: string): Chainable<Element>;
       checkApplicationStatus(appName: string, clusterName?: string): Chainable<Element>;
       deleteApplicationDeployment(clusterName?: string): Chainable<Element>;
+      modifyDeployedApplication(appName: string,clusterName?: string): Chainable<Element>;
```

Comment:
Space after the comma on `string,` would be nice.
It can be done in future.

Reply:
I will do it in follow-up PR. Thanks for pointing out.